lib/resouces/app_colors.dart

```diff
@@ -0,0 +1,10 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_survey/utils/color_util.dart';
+
+class AppColors {
+  AppColors._();
+
+  static Color whiteAlpha18 = ColorUtil.fromHex("#2EFFFFFF");
+  static Color whiteAlpha50 = ColorUtil.fromHex("#80FFFFFF");
```

Comment:
alternatively: `Colors.white.withAlpha(18)` to avoid static footprint? 🤔  as I think `AppColors` now contains all the colors that are actually also provided via the native `Colors` - so it's kinda redundant somehow? 🤔 

Unless we keep a ref like: `static Color primaryColor = Colors.white.withAlpha(50)` then I think it makes sense

Reply:
I ended up by defining a colors xml resource file instead https://github.com/luongvo/flutter-survey/pull/42/commits/48dcf67a22d8a3c6781ff83d9c645e6f1d25c18b#diff-203c94c4eed1defcbda5fcbf310a575a9dc273419dd270446f0322818af8b183 🏃